modules/core/src/v2/internal/tssUtils.ts

```diff
@@ -285,4 +340,153 @@ export class TssUtils {
 
     return keychains;
   }
+
+  async signTSSTxRequest(
```

Comment:
maybe rename to `signAndSendTxRequest`
can drop TSS since its implied - function's in `tssUtils`